src/main/java/homework_1/ConsolePrinter.java

```diff
@@ -0,0 +1,22 @@
+package homework_1;
+
+import ConsoleColors.ConsoleColors;
+
+public class ConsolePrinter {
+    public static void main(String[] args) {
+
+        if (args.length > 0) {
+            for (int i = 0; i < args.length; i++) {
+
+                String toPrint = args[i];
+                if (toPrint.matches("error")) {                                //checking for error
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/lang/String.html#matches(java.lang.String)

i think it's more efficient to use equalsIgnoreCase()